src/katsdpcontroller/generator.py

```diff
@@ -1010,6 +1010,8 @@ def get_baseline_index(a1, a2):
                     "Number of visibilities that saturated",
                     name_regex=re.compile(rf"{escaped_name}\.[0-9]+\.xeng-clip-cnt"),
                     n_children=stream.n_substreams,
+                    auto_strategy=SensorSampler.Strategy.EVENT_RATE,
+                    auto_strategy_parameters=(FAST_SENSOR_UPDATE_PERIOD, math.inf),
```

Comment:
I didn't end up going with `**FAST_SENSOR_UPDATE_PARAMS` because
* mypy threw all sorts of complaints about receiving a `Dict[str, object]` (and I did try debugging for a bit), and
* I've reduced it to two usages of the kwargs.

Still, let me know if I should persevere.